package/MDAnalysis/visualization/streamlines_3D.py

```diff
@@ -281,15 +281,15 @@ def update_dictionary_point_in_cube_start_frame(array_simulation_particle_coordi
                                                            axis=0)
                 cube['centroid_of_particles_first_frame'] = centroid_particles_in_cube
             else:  # empty cube
-                cube['centroid_of_particles_first_frame'] = 'empty'
+                cube['centroid_of_particles_first_frame'] = None
             cube_counter += 1
 
     def update_dictionary_end_frame(array_simulation_particle_coordinates, dictionary_cube_data_this_core):
         """Update the cube dictionary objects again as appropriate for the second and final frame."""
         cube_counter = 0
         for key, cube in dictionary_cube_data_this_core.items():
             # if there were no particles in the cube in the first frame, then set dx,dy,dz each to 0
-            if cube['centroid_of_particles_first_frame'] == 'empty':
+            if cube['centroid_of_particles_first_frame'] is None:
```

Comment:
comment: Looked innocently enough...

Reply:
I'm surprised it's the only failure we have! 1.25 is going to cause a lot of trouble for legacy downstream codes.

Reply:
there's a really similar 'empty' check in https://github.com/MDAnalysis/mdanalysis/blob/0c4f823a457fd9ca91c25e41b318b3b2d0c582d8/package/MDAnalysis/visualization/streamlines.py#L204

Those probably also need to get changed...

Reply:
Can confirm the same FutureWarning is happening in streamlines :/ the test isn't good enough to pick it up though.